cartographer/mapping/3d/submap_3d.cc

```diff
@@ -211,6 +214,12 @@ Submap3D::Submap3D(const proto::Submap3D& proto)
           common::make_unique<HybridGrid>(proto.low_resolution_hybrid_grid())) {
   set_num_range_data(proto.num_range_data());
   set_finished(proto.finished());
+  rotational_scan_matcher_histogram_.resize(
```

Comment:
Eigen says 'resizing a matrix to 0 is not supported'. 
Can you either handle this here (not sure if it should or can happen) or CHECK(proto.rotational_scan_matcher_histogram_size())?